tracer/build/supported_versions.json

```diff
@@ -88,6 +88,23 @@
     "maxAssemblyVersionInclusive": "9.65535.65535",
     "packages": []
   },
+  {
+    "integrationName": "AspNetCore",
+    "assemblyName": "Microsoft.AspNetCore.Http.Abstractions",
```

Comment:
Unsure why this has appeared 🤔

Reply:
AFAICT, instrumentation of this assembly was added in #6733